src/cdomains/apron/elinaImplementation.elina.ml

```diff
@@ -0,0 +1,154 @@
+open Apron
+
+module ElinaImplementation = struct
+  (*
+  This is supposed to implement the same functionality as Abstract1.substitute_texpr_with
+  which for some reason doesn't seem to work correctly in elina?

```

Comment:
>which for some reason doesn't seem to work correctly in elina?

Could you clarify what the problem is, and ideally link to the issue, so one can check later whether it has been fixed in the meantime?

Reply:
Added the link to the issue